v1/routes/breweries.py

```diff
@@ -7,9 +7,36 @@
 router = APIRouter()
 
 
-@router.get("/")
-async def endpoint():
-    """
-    Ping style health check
-    """
-    return {"endpoint": "breweries"}
+@router.get("/", status_code=200)
+async def get_all_breweries():
+    return {"endpoint": "get_all_breweries"}
+
+
+@router.post("/", status_code=201)
+async def post_breweries():
+    return {"endpoint": "post_breweries"}
+
+
+@router.get("/random/", status_code=200)
+async def get_random_brewery():
+    return {"endpoint": "get_random_brewery"}
+
+
+@router.get("/meta/", status_code=200)
+async def meta():
+    return {"endpoint": "metadata"}
+
+
+@router.delete("/{id}/", status_code=501)
+async def delete_brewery_by_id():
+    return {"endpoint": "delete_brewery_by_id"}
```

Comment:
Let's follow the current API and return a 404 for `DELETE`. I also think this can point to the root route `/`.

<img width="1419" alt="Screenshot 2023-01-21 at 6 33 26 PM" src="https://user-images.githubusercontent.com/96110/213898040-1325276e-845a-48dc-becd-bf3ea07f5b92.png">

Reply:
Done.